src/components/Navbar.tsx

```diff
@@ -8,15 +11,26 @@ export const Navbar = () => {
     >
       <div className="container">
         <div className="navbar-brand">
-          <a className="navbar-item" href="#/">Home</a>
+          <NavLink
+            to="/"
```

Comment:
className prop is receiving a function that returns a string, but it should receive a string directly. Consider extracting the function and making it return a string.